api/aperture/policy/language/v1/autoscale.proto

```diff
@@ -134,6 +134,18 @@ message DecreasingGradient {
   Parameters parameters = 2;
 }
 
+// PeriodicDecrease defines a controller for scaling in based on a periodic timer.
+message PeriodicDecrease {
+  message Parameters {
+    // The period of the timer.
+    google.protobuf.Duration period = 1; // @gotags: default:"1m"
+    // The percentage of replicas to scale in.
```

Comment:
This controller is independent of Kubernetes so use a generic language like "percentage of scale to reduce"

Reply:
Addressed in [0d12ad9](https://github.com/fluxninja/aperture/pull/1923/commits/0d12ad9a06acab38385913cff7753406989a3211)